### DISCUSSION TITLE
Error return value of `(*github.com/gofiber/fiber/v2.Ctx).JSON` is not checked (errcheck)

### DISCUSSION THREAD

**Discussion by raliqala** [golangci/golangci-lint#2376] [Q&A] [answered]
Hi everyone  

I'm getting this error when I run ```golangci-lint run```   
```
Error return value of `(*github.com/gofiber/fiber/v2.Ctx).JSON` is not checked (errcheck)
```
this error is good but its a difficult problem to solve because in my code I return with a specific status, here is my code  
```
	return c.Status(fiber.StatusBadRequest).JSON(fiber.Map{
			"success": false,
			"error": fiber.Map{
				"password": err.Error(),
			},
		})
```   
I saw a solution here https://github.com/gofiber/fiber/issues/126#issuecomment-585082611  

but this solution is hard to implement in my code.  

Please help  
Thanks in advance

**Comment by ldez** [comment, marked as answer]
Hi,

You have 3 solutions:
1. explicitly ignore the error
```go
_ = c.Status(fiber.StatusBadRequest).JSON(fiber.Map{
	"success": false,
	"error": fiber.Map{
		"password": err.Error(),
	},
})
```
2. Add an exclusion:
```yml

linters-settings:
  errcheck:
    exclude-functions:
      - (*github.com/gofiber/fiber.Ctx).JSON
```
3. handle the error
```go
err = c.Status(fiber.StatusBadRequest).JSON(fiber.Map{
	"success": false,
	"error": fiber.Map{
		"password": err.Error(),
	},
})
if err != nil {
	c.Next(err)
}
```

**Reply by raliqala** [reply]
Thank you so Mach for answering.   

Thank you
